surefire-api/src/main/java/org/apache/maven/surefire/util/DefaultRunOrderCalculator.java

```diff
@@ -140,4 +203,10 @@ public int compare( Class o1, Class o2 )
             }
         };
     }
+
+    public ConsoleLogger getConsoleLogger()
```

Comment:
If this returns /dev/null, why even bother making such a method, or use a logger whatsoever?